prospect/sources/galaxy_basis.py

```diff
@@ -414,14 +418,16 @@ def one_sed(self, component_index=0, filterlist=[]):
                 self.csp.params[k] = vv
             if k == 'mass':
                 mass = v
-        # Now get the magnitudes and spectrum
+        # Now get the magnitudes and spectrum.  The spectrum is in units of
+        # Lsun/Hz/per solar mass *formed*
         w, spec = self.csp.get_spectrum(tage=self.csp.params['tage'], peraa=False)
-        mags = getSED(w , lightspeed/w**2 * spec * to_cgs, filterlist)
-        #mags = self.csp.get_mags(tage=self.csp.params['tage'], bands=filterlist)
-        # Normalize by (current) stellar mass and get correct units
-        cmass = self.csp.stellar_mass
-        mass_norm = mass / cmass
-        return mass_norm * spec, mass_norm * 10**(-0.4*(mags)), cmass
+        mags = getSED(w, lightspeed/w**2 * spec * to_cgs, filterlist)
+        mfrac = self.csp.stellar_mass
+        if np.all(self.params.get('mass_units', 'mstar') == 'mstar'):
+            # Convert normalization units from per stellar masss to per mass formed
```

Comment:
"# Convert normalization units from per stellar masss to per mass formed"

Should this be from mass formed to stellar mass?